documentation/openapi.yaml

```diff
@@ -0,0 +1,119 @@
+openapi: 3.0.0
+info:
+  version: 1.0.0-oas3
+  title: Kubportal Backend API
+paths:
+  /statistics/{stat}:
+    get:
+      parameters:
+        - in: path
+          name: stat
+          required: true
+          schema:
+            type: string
+          description: The statistic type being fetched
+      responses:
+        '200':
+          description: A single statistic value
+          content:
+            application/json:
+              schema:
+                type: object
```

Comment:
Why is this an object, and not a simple string? Is there any chance to have a single statistic with multiple key-value-pairs?

Reply:
no, I don't have any arguments. I'll change it.